src/utils/runConfigs.js

```diff
@@ -55,6 +55,13 @@ export const DockerRun = {
       input_type: 'object',
       label: 'Environment variables',
       description: 'Additional environment variables to set in the container.'
+    },
+    {
+      arg: 'host_config',
+      input_type: 'object',
+      label: 'Host Config',
+      description:
+        'A dictionary or runtime arguments to pass to the Docker Agent. See link below for options.'
```

Comment:
Where's the link?

Reply:
Oops, didn't want to include a link there, updated 👍🏻